doc/arkode/guide/source/Usage/ERKStep/User_callable.rst

```diff
@@ -33,6 +33,10 @@ ERKStep supports the following categories:
 * temporal adaptivity
 * relaxation Runge--Kutta methods
 
+ERKStep does not have forcing function support when converted to a
```

Comment:
Similarly, this will need updating: https://github.com/LLNL/sundials/blob/365515a4c955c52864a333a9c2ba36f6dc28a317/doc/arkode/guide/source/Usage/User_callable.rst?plain=1#L4905-L4908